crowbar_framework/spec/controllers/crowbar_controller_spec.rb

```diff
@@ -97,7 +97,7 @@
     end
 
     it "returns node as a hash on success when passed a node (backward compatibility)" do
-      CrowbarService.any_instance.stubs(:transition).returns([200, NodeObject.find_node_by_name("testing").to_hash])
+      allow_any_instance_of(CrowbarService).to receive(:transition).and_return([200, NodeObject.find_node_by_name("testing").to_hash])
```

Comment:
Metrics/LineLength: Line is too long. [134/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)